docs/source/merlin_commands.rst

```diff
@@ -235,47 +235,44 @@ with workers, such as via ``merlin stop-workers --workers``.
 
 .. _stop-workers:
 
-Stopping workers (stop-workers)
--------------------------------
+Stopping workers (``merlin stop-workers``)
+------------------------------------------
```

Comment:
This also needs --task_server

Reply:
The reason I excluded task server is that the way merlin currently works, the only possible task server is celery. So I thought removing that choice from the docs might help users from getting confused.

(Any opinion on this @lucpeterson?)

Reply:
Yes, that is a good reason.  I only mention it for completeness, if they have a  question about the arguments for ``merlin stop-workers -h`` where will they that information?

Reply:
What I'll do for now is add a section about this, and mention that celery is the only option. We might consider deprecating this flag later on.

Reply:
Sounds good.